ci/docker/install/centos7_install_ccache.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Script to build ccache for centos7 based images
+
+set -ex
+
+pushd .
+
+yum -y install epel-release
+yum -y install git
+yum -y install autoconf
+yum -y install wget
+yum -y install make
+yum -y install google-perftools
+yum -y install asciidoc
+yum -y install gcc-c++-4.8.*
+
+mkdir -p /work/deps
+cd /work/deps
+
+git clone --recursive -b v3.4.2 https://github.com/ccache/ccache.git
+
+cd ccache
+
+./autogen.sh
+./configure
+make -j$(nproc)
+make install
+
+cd /work/deps
+rm -rf /work/deps/ccache
+
+popd
+
+export CCACHE_MAXSIZE=${CCACHE_MAXSIZE:=10G}
```

Comment:
who will pick up these variables? Shouldn't it be in the dockerfile or in the python script?

Reply:
I was thinking about it but decided to keep it here - it has the advantage to configure what was installed in 1 place. And actually this is setting only a default value if variable was not set before by the script.

Reply:
I think we should set it in build.py, considering we're bootstrapping ccache there anyways and it allows a centralized configuration. With the current way, you'd have to change it in multiple scripts (ubuntu, centos, windows, arm, etc)

Reply:
I don't think this variable gets exported outside of this script. So it won't be used. Am I wrong?

Reply:
In the current way you change it only in this script. And the way it's done does not prevent you to have it in build.py as well.

Reply:
The problem is that this environment variable will not persist, means this call is a NO-OP.

Reply:
Yes, I agree that might be a problem